Oryx.Cognite/src/Playground/Annotations.fs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+namespace Oryx.Cognite.Playground
+open System.Collections.Generic
+
+open Oryx
+open Oryx.Cognite
+open Oryx.Cognite.Playground
+
+open CogniteSdk
+
+/// Various annotation HTTP handlers.
+
+[<RequireQualifiedAccess>]
+module Annotations =
+    [<Literal>]
+    let Url = "/annotations"
+
+    /// <summary>
+    /// Create new annotations in the given project.
+    /// </summary>
+    /// <param name="annotations">The annotations to create.</param>
+    /// <returns>List of created annotations.</returns>
+    let create (annotations: AnnotationCreate seq) (source: HttpHandler<unit>) : HttpHandler<Annotation seq> =
+        source
+        |> withLogMessage "Annotation:create"
+        |> create annotations Url
+
+    /// <summary>
+    /// Delete multiple annotations in the same project.
+    /// </summary>
+    /// <param name="items">The list of ids for Annotations to delete.</param>
+    /// <returns>Empty result.</returns>
+    let delete (items: AnnotationDelete) (source: HttpHandler<unit>) : HttpHandler<EmptyResponse> =
+        source
+        |> withLogMessage "Annotations:delete"
+        |> delete items Url
+
+    /// <summary>
+    /// Retrieves list of annotations matching filter, and a cursor if given limit is exceeded
+    /// </summary>
+    /// <param name="query">The query to use.</param>
+    /// <returns>List of annotations matching given filters and optional cursor</returns>
+    let list (query: AnnotationQuery) (source: HttpHandler<unit>) : HttpHandler<ItemsWithCursor<#Annotation>> =
+        source
+        |> withLogMessage "Annotations:list"
+        |> list query Url
+
+
```

Comment:
Remove extra newline

Reply:
fixed in [8a6f4ca](https://github.com/cognitedata/cognite-sdk-dotnet/pull/287/commits/8a6f4ca65ad825bfa2d79a3230cc4ddc834da8e1)